Tools/Towel_Testing/DataStructures/AvlTree.cs

```diff
@@ -4,12 +4,15 @@
 using Towel;
 using static Towel.Statics;
 using Towel.DataStructures;
+using System.Collections.Generic;
 
 namespace Towel_Testing.DataStructures
 {
-	[TestClass] public class AvleTreeLinked_Testing
+	[TestClass]
```

Comment:
These are an example of me not following C# conventions (put attribute above the method/class). I kinda like the attribute to the left of the method/class, but it is going to cause too many issues since auto-formatting flips it...

So going forward... I'll start following C# conventions here to make it easier on everyone and start putting attributes on methods/classes above them.

Reply:
Don't know much about StyleCop, but is it possible to add such a formatting option? My code Formatter is doing it automatically...